dockers/docker-database/Dockerfile.j2

```diff
@@ -60,4 +62,6 @@ COPY ["critical_processes", "/etc/supervisor"]
 COPY ["files/update_chassisdb_config", "/usr/local/bin/"]
 COPY ["flush_unused_database", "/usr/local/bin/"]
 
+LABEL com.azure.sonic.manifest="$manifest"
```

Comment:
A thought:
Can we add a check in build_debian.sh, to see that every docker pulled in does have this label?

Reply:
Yes, we can, thanks for suggestion. Please check the last commit